src/hotspot/share/gc/serial/tenuredGeneration.cpp

```diff
@@ -440,7 +440,15 @@ void TenuredGeneration::verify() {
 }
 
 void TenuredGeneration::print_on(outputStream* st)  const {
-  Generation::print_on(st);
-  st->print("   the");
+  st->print(" %-10s", name());
+
+  st->print(" total " SIZE_FORMAT "K, used " SIZE_FORMAT "K",
+            capacity()/K, used()/K);
+  st->print_cr(" [" PTR_FORMAT ", " PTR_FORMAT ", " PTR_FORMAT ")",
+               p2i(_virtual_space.low_boundary()),
+               p2i(_virtual_space.high()),
+               p2i(_virtual_space.high_boundary()));
+
+  st->print("  the");
```

Comment:
The new `st->print("  the");` (new line 452) misses one space. Please note the old line (old line 444) has three spaces. Is it your intention?

Reply:
It was intentional, to make sure `the` is not indented too much. Now that I realize that using three-space was to align "space" in those log line. Reverted.